src/csvcubed/readers/skoscodelistreader.py

```diff
@@ -12,10 +12,8 @@
 from uritemplate import variables
 
 from csvcubed.utils.iterables import first
-from csvcubed.utils.sparql_handler.sparqlquerymanager import (
-    select_table_schema_properties,
-)
-from csvcubed.utils.tableschema import CsvwRdfManager
+from csvcubed.utils.sparql_handler.code_list_inspector import CodeListInspector
+from tests.helpers.inspectors_cache import get_csvw_rdf_manager
```

Comment:
This test code should not be used anywhere outside tests. The cache is *only* for testing.